app/models/student.rb

```diff
@@ -16,6 +16,42 @@ def assign_user(current_user)
     end
   end
 
+  def owes_money?(month)
+    dates = MeetingDate.where('extract(month from third) = ?', month)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.